src/plugins/displayLayout/DisplayLayoutToolbar.js

```diff
@@ -0,0 +1,130 @@
+define([], function () {
+
+    function DisplayLayoutToolbar(openmct) {
+        return {
+            name: "Display Layout Toolbar",
+            key: "layout",
+            description: "A toolbar for objects inside a display layout.",
+            forSelection: function (selection) {
+                // Apply the layout toolbar if the selected object is inside a layout,
+                // and in edit mode.
+                return (selection &&
+                    selection[1] &&
+                    selection[1].context.item &&
+                    selection[1].context.item.type === 'layout' &&
+                    openmct.editor.isEditing());
+            },
+            toolbar: function (selection) {
+                let domainObject = selection[1].context.item;
+                let layoutItem = selection[0].context.layoutItem;
+
+                if (layoutItem && layoutItem.type === 'telemetry-view') {
+                    let path = "configuration.alphanumerics[" + layoutItem.config.alphanumeric.index + "]";
+                    let metadata = openmct.telemetry.getMetadata(layoutItem.domainObject);
+
+                    return [
+                        {
+                            control: "select-menu",
+                            domainObject: domainObject,
+                            property: path + ".displayMode",
+                            title: "Set display mode",
+                            options: [
+                                {
+                                    name: 'Label + Value',
+                                    value: 'all'
+                                },
+                                {
+                                    name: "Label only",
+                                    value: "label"
+                                },
+                                {
+                                    name: "Value only",
+                                    value: "value"
+                                }
+                            ]
+                        },
+                        {
+                            control: "separator"
+                        },
+                        {
+                            control: "select-menu",
+                            domainObject: domainObject,
+                            property: path + ".value",
+                            title: "Set value",
+                            options: metadata.values().map(value => {
```

Comment:
If metadata is with void values, maybe a filter would be better?

Reply:
metadata is a TelemetryMetadataManager not an Array.

Reply:
@evenstensberg all the values that are returned by values() should be used to create the name value pair and none should be filtered.